stage.sh

```diff
@@ -12,7 +12,7 @@ BUCKET=$1
 S3PATH=$2
 
 # remove this line if you want to keep the objects private in your S3 bucket
-export S3PUBLIC=" --acl public-read"
+#export S3PUBLIC=" --acl public-read"
```

Comment:
I think you missed to uncomment this piece

Reply:
Yep. Good point.